System/Helper.gd

```diff
@@ -0,0 +1,14 @@
+extends Node
```

Comment:
- Perhaps this could be System/WrightScript/Parsing.gd?
- It should extend Reference if it is not meant to be instantiated into a Godot Scene (Reference objects are reference counted, while Node objects are freed when they are removed from a SceneTree)